packages/stores/src/mockStore.ts

```diff
@@ -5,11 +5,28 @@ const mock = (): TStores & { is_mock: boolean } => {
     return {
         is_mock: true,
         client: {
+            fetchResidenceList: jest.fn(),
+            fetchStatesList: jest.fn(),
+            getChangeableFields: jest.fn(),
+            residence_list: [
+                {
+                    text: 'Text',
+                    value: 'value',
+                },
+            ],
+            states_list: [
+                {
+                    text: 'Text',
+                    value: 'value',
+                },
+            ],
             account_settings: {},
             accounts: {},
+            is_social_signup: false,
             active_account_landing_company: '',
             trading_platform_available_accounts: [],
             account_limits: {
```

Comment:
```suggestion
            account_limits: {},
```
https://github.com/binary-com/deriv-app/blob/master/packages/core/src/Stores/client-store.js#L134

Not sure why we had data for it before 🤔 Maybe it was missed in code review.